tests/test_graphic_walker.py

```diff
@@ -1,7 +1,10 @@
-import numpy as np
+import json
```

Comment:
The pytests on the main branch are now failing.

![image](https://github.com/user-attachments/assets/426633c8-68b0-43af-8048-9b6829e1c628)